sinks_output_test.go

```diff
@@ -24,88 +24,114 @@ func TestOutputSink_Check(t *testing.T) {
 
 	logs := `
 foo debug
+foo info
 foo error
 foo.bar debug
+foo.bar info
 foo.bar error
 foo.bar.baz debug
+foo.bar.baz info
 foo.bar.baz error
 foo.bar.baz1 debug
+foo.bar.baz1 info
 foo.bar.baz1 error
 `
 
 	cases := []struct {
 		Name string
 		Env  map[string]string
-		Want string
+		Want autogold.Value
```

Comment:
minor: I somewhat dislike the introduction of autogold here. These are not normal snapshot tests, if you are changing behaviour you need to be explicit and not just rely on `go test -update`.